src/content-helper/excerpt-generator/excerpt-generator.tsx

```diff
@@ -44,8 +44,13 @@ const ExcerptGenerator = ( settings: never, name: string ) => {
 		render: ExcerptPanel,
 	} );
 
-	// Remove the excerpt panel by dispatching an action.
-	dispatch( 'core/edit-post' )?.removeEditorPanel( 'post-excerpt' );
+	/* Remove the excerpt panel by dispatching an action. */ // @ts-ignore
```

Comment:
Just wondering why we need the extra `@ts-ignore` on this line, since it's a comment.

Reply:
We had two TypeScript errors, the first one being in the `if` statement. `ts-ignore` influences the next line, so it has been added to cater for that. I could have added it in a new line but preferred this approach which was the only one that was working. Adding it in the end of a `//` comment didn't work, it needs to be its own comment.

Unfortunately, it's not possible to ignore code blocks using `ts-ignore`, so this was the best way I found to ignore the errors and make the code look OK.

Reply:
Ahhhhh gotcha! I have dealt with that issue myself a few times as well. Thanks for explaining :)